### DISCUSSION TITLE
Png to dxf

### DISCUSSION THREAD

**Discussion by dev-hack95** [mozman/ezdxf#926] [General]
I there any way to convert png to dxf files

**Comment by mozman** [comment]
No, that requires shape recognition in images, maybe [opencv](https://pypi.org/project/opencv-python/) can help you.

**Reply by dev-hack95** [reply]
1. Yes we have use dfs algorithum using opencv but it stuck in a loop for circle and line crossosing in unable to detect
2. Can you have a code regrading converting cartesian co-ordinates to dxf formats it will be very use if we can define this kind of things if it is possible we can map cartesian points and convert them into dxf formats

**Comment by dev-hack95** [comment]
3. Or any deep learning technique that will detect shape and plot co-ordinates using ezdxf for that also we have to know how this will work
